stable/insights-agent/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart to run the Fairwinds Insights agent
 name: insights-agent
-version: 0.19.1
+version: 0.19.2
```

Comment:
Can we make this `0.20.0`? Usually if we're adding new features we bump the minor version.

Reply:
Done.